client/src/components/CompareDay/CompareDay.ts

```diff
@@ -0,0 +1,9 @@
+export const compareDay = (day1: Date, day2: Date): boolean => {
+  if (
+    day1.getFullYear() == day2.getFullYear() &&
+    day1.getMonth() == day2.getMonth() &&
+    day1.getDate() == day2.getDate()
```

Comment:
Could this be shortened to just one line with `Date.valueOf()`?

Reply:
Yeah I initially tried to use Date.getTime(), but I didn't know you can format the output directly as you mentioned in another comment, so it seemed like more work.